clients/drcachesim/docs/drcachesim.dox.in

```diff
@@ -1147,6 +1147,8 @@ $ bin64/drrun -t drmemtrace -indir drmemtrace.app.pid.xxxx.dir/
 
 The direct results of the \p -offline run are raw, compacted files, stored
 in a \p raw/ subdirectory of the \p drmemtrace.app.pid.xxxx.dir directory.
+The contents of \p drmemtrace.app.pid.xxxx.dir, and all its subdirectories,
```

Comment:
nit: missing "the": s/of/ of the/
(be consistent with all the other references such as line 1149 and line 1154 which have "the")

Reply:
I'd expect the sole addition of "the" here to be bad grammar.
The other instances you point out say "... of the drmemtrace.app.pid.xxxx.dir directory",
whereas here the wording is "... of drmemtrace.app.pid.xxxx.dir".
Shall I add "directory" as well ? [gotta get the grammar correct! :-)]

i.e., "The contents of the \p drmemtrace.app.pid.xxx.dir directory, and all its subdirectories, ..."

Reply:
Ah agreed, must have read it as the same as the others: good as-is or with "the ... directory".